demos/keyboard_nav/index.html

```diff
@@ -58,18 +83,26 @@ <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
   </p>
 
   <p>
-      Enable Accessibility Mode:
-      <input type="checkbox" onclick="toggleAccessibilityMode(this.checked)" id="accessibilityModeCheck">
-      <select id="cursorChanger" name="cursor" onchange="changeCursor(this.value)">
-        <option value="default">Default Cursor</option>
-        <option value="basic">Basic Cursor</option>
-      </select>
+    Enable Accessibility Mode:
```

Comment:
could be nice to also put this text in a label (like of open key mappings)

Reply:
done